.gitignore

```diff
@@ -102,6 +102,7 @@ imperative_debug.log
 # Licenses in package folders (managed with build scripts)
 packages/*/LICENSE
 zowex/
+packages/secrets/
```

Comment:
thanks for gitignoring this directory. 
it was kind of a pain to have to remove it whenever you switch back and forth  between v2 and v1 😋